engine/class_modules/sc_death_knight.cpp

```diff
@@ -2666,6 +2674,106 @@ struct magus_pet_t : public death_knight_pet_t
   }
 };
 
+struct reanimated_shambler_pet_t : public death_knight_pet_t
+{
+  struct necroblast_t : public pet_action_t<reanimated_shambler_pet_t, melee_attack_t>
+  {
+    necroblast_t( reanimated_shambler_pet_t* player, util::string_view options_str ) :
+      pet_action_t( player, "necroblast", player -> find_spell( 334851 ) )
+    {
+      parse_options( options_str );
```

Comment:
necroblast is a proc action, it shouldn't use (or have access to) options

Reply:
I thought things executed by the APL use options_str.  Does it not count here because the shambler apl is fixed?

Reply:
Well it's only necessary if you plan to add options to necroblast in the shambler APL.